validation/standard/standard-versions-sepolia.toml

```
@@ -3,6 +3,25 @@
 #   * proxied             : specify a standard "implementation_address"
 #   * neither             : specify neither a standard "address" nor "implementation_address"
 
+# OPCM https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv2.0.0-rc.1
+["op-contracts/v2.0.0-rc.1"]
+# Unchanged in this release
+system_config = { version = "2.4.0", implementation_address = "0x760C48C62A85045A6B69f07F4a9f22868659CbCc" }
+fault_dispute_game = { version = "1.4.1" }
+permissioned_dispute_game = { version = "1.4.1" }
+mips = { version = "1.3.0", address = "0xaa59a0777648bc75cd10364083e878c1ccd6112a" }
+optimism_portal = { version = "3.13.0", implementation_address = "0x2D7e764a0D9919e16983a46595CfA81fc34fa7Cd" }
+anchor_state_registry = { version = "2.2.2" }
```

The ASR is now MCP ready, so we can probably start including it here
```suggestion
anchor_state_registry = { version = "2.2.2",  implementation_address = "0x7b465370BB7A333f99edd19599EB7Fb1c2D3F8D2" }
```